### ISSUE TITLE
Akhil Review Ticket - Week 11

### ISSUE THREAD

**Issue by AkhilNandhakumar** [AkhilNandhakumar/Team-MicrosoftTechSupport#61]
# Score Given By Corssover (Tigran A.): 9.8 / 10
---------------------------
## Individual Grading:
* [Wiki/Jekyll page with design ideas and usage](https://github.com/YashShah138/Team-MicrosoftTechSupport/wiki/PBL-Create-Task-Dry-Run#akhil---site-search-feature---video-link) **1 / 1**
* [Create Performance Task PBL feature](https://github.com/YashShah138/Team-MicrosoftTechSupport/wiki/PBL-Create-Task-Dry-Run#3a-1) **1 / 1**
* [Final Commited Program Code](https://github.com/YashShah138/Team-MicrosoftTechSupport/blob/ee14dfae0d25b1812d7c171747f12bcc4982f839/templates/layouts/base.html#L44-L112) **1 / 1**
* [1 minute video](https://youtu.be/98KXo33uTuY) **0.9 / 1** (organization and ease of finding info)
### Individual total: 4.9 / 5
---------------------------
## Team Grading:
[PBL Github pages:Jekyll, WIKI, live review focus](https://github.com/YashShah138/Team-MicrosoftTechSupport/wiki/PBL-FINAL-OVERVIEW) 
* [PBL theme consistent through project and described in Github Pages](https://github.com/YashShah138/Team-MicrosoftTechSupport/wiki/PBL-FINAL-OVERVIEW#website-theme) **0.5 / 0.5**
* [Sassy / Bootstrap](https://github.com/YashShah138/Team-MicrosoftTechSupport/wiki/PBL-FINAL-OVERVIEW#sassybootstrap) **0.5 / 0.5**
* [Blueprint illustration/demonstration](https://github.com/YashShah138/Team-MicrosoftTechSupport/wiki/PBL-FINAL-OVERVIEW#blueprint-demonstration) **0.5 / 0.5**
* [CRUD Async Operation](https://github.com/YashShah138/Team-MicrosoftTechSupport/wiki/PBL-FINAL-OVERVIEW#crud-async-operation-demonstrated-in-runtime-video) **0.5 / 0.5**
* [Runtime and understanding of JavaScript and Fetch](https://youtu.be/ASsK6VMMhAA?t=28) **0.5 / 0.5**
* [Individual PBL MVC feature/commits showing algorithms written in JS/Python](https://github.com/YashShah138/Team-MicrosoftTechSupport/wiki/PBL-FINAL-OVERVIEW#features-and-commits) **0.5 / 0.5**
* [Highlight of best features **0:00 - 0:28**](https://www.youtube.com/watch?v=ASsK6VMMhAA) **0.5 / 0.5**
* [CRUD/Search on project database tables, easy to identify from UI **0:20 - 0:39**](https://youtu.be/ASsK6VMMhAA?t=20) **0.5/0.5**
* [MVC - a code overview showing usage of MVC **0:39 - 0:54**](https://youtu.be/ASsK6VMMhAA?t=39) **0.5 / 0.5**
* [Google Search or Google translate implemented **0:58 - 1:00** ](https://youtu.be/ASsK6VMMhAA?t=58) **0.5 / 0.5**
* [Scrum Board - Overview of completed tasks and Github tangibles **0:54 - 1:03** ](https://youtu.be/ASsK6VMMhAA?t=54) **0.5/0.5**

**- 0.1 point for clarity of finding info:** Video is very fast and a lot of pausing is needed to read subtitles, google translate and code commits inserted into the video in awkward spots.
### Team Section Total: 4.9 / 5
-------------
_Comments from Tigran Below this in comment thread_

**Comment by Tigran7** [comment]
AKHIL'S FINAL SCORE: 9.8/10

-.1 For clarity of his Create task video
-.1 For clarity of his PBL video

- I liked the idea of Akhil and his group and I think they executed well on their idea.
-  The team in general made a website relating to information on our local beaches, while Akhil made a search bar for the content found on his website.
- He provided tangibles for everything and overall everything is well organized on the pages that he linked.

**Individual/Pair Evaluation on Create Task Project - [4.9/5]**
- Wiki/Jekyll page with design ideas and usage
[ 1/1] All the information is on a Wiki page, design ideas and usage included on the bottom of the page.
- Create Performance Task PBL feature
[ 1/1] The beach theme of the PBL page is clearly met
- Final committed program code showing algorithms written in JS/Python
[1/1] A link to the final committed code is included on the wiki he linked
- Written responses to all the prompts in the performance task
1/1] All prompts are answered with sufficient detail and explaination
- 1min video that meets CB create task requirements
[ .9/1] His create task video shows his work but its a little bit hard to follow

**Team Final Grading Considerations -[4.9/5]**
**PBL Github pages. Jekyll, WIKI, live review focus [2.5/2.5]**
- PBL theme consistent through project and described in Github Pages
[.5/.5] The group has a clear beach theme throughout the video
- Sassy / Bootstrap
[.5/.5] Website includes sassy/bootstrap 
- Blueprint illustration/demonstration
[.5/.5] Blueprinting is clearly implemented in their code
- CRUD Async Operation - runtime and understanding of JavaScript and Fetch
[.5/.5] Async is fully functional as well as the CRUD database
- Individual PBL MVC feature/commits showing algorithms written in JS/Python
[.5/.5] Example of MVC given in PBL
**Runtime Features shown in 1 min video [2.4/2.5]**
- Highlight of best features
[.4/.5] Best features are highlighted in the video but it's a little bit choppy and hard to follow
- CRUD/Search on project database tables, easy to identify from UI
[.5/.5] The CRUD is shown and easy to identify
- MVC - a code overview showing usage of MVC (HTML, Routes/Render Template functions, Model functions)
[.5/.5] MVC is shown in the video as well as code for it
- Google Search or Google translate implemented
[.5/.5] Google translate is implemented and fully-functional
- Scrum Board - Overview of completed tasks and Github tangibles
[.5/.5] The scrum board is shown in full in the video with all the tasks from each week
